sdk/core/core-tracing/review/core-tracing.api.md

```diff
@@ -5,183 +5,116 @@
 ```ts
 
 // @public
-export interface Context {
-    deleteValue(key: symbol): Context;
-    getValue(key: symbol): unknown;
-    setValue(key: symbol, value: unknown): Context;
-}
-
-// @public
-const context_2: ContextAPI;
-export { context_2 as context }
-
-// @public
-export interface ContextAPI {
-    active(): Context;
-}
+export function createTracingClient(options: TracingClientOptions): TracingClient;
 
 // @public
-export function createSpanFunction(args: CreateSpanFunctionArgs): <T extends {
-    tracingOptions?: OperationTracingOptions | undefined;
-}>(operationName: string, operationOptions?: T | undefined, startSpanOptions?: SpanOptions | undefined) => {
-    span: Span;
-    updatedOptions: T;
-};
+export function createTracingContext(options?: CreateTracingContextOptions): TracingContext;
 
 // @public
-export interface CreateSpanFunctionArgs {
-    namespace: string;
-    packagePrefix: string;
+export interface CreateTracingContextOptions {
+    client?: TracingClient;
+    namespace?: string;
+    parentContext?: TracingContext;
```

Comment:
Sorry if I missed something: what's the relationship between `parentContext` and `span` here?

Reply:
A `parentContext` is the existing context to use for any initial values. The new context will contain all the initial values.

in addition, there are a few "known" values you likely will want to set on a context - for example a span. In OTel you'd call `setSpan` to achieve this. Here you would pass the new context in when creating a context. 

We don't have to have this here, its just a convenience for well-known code paths. Alternatively `createTracingContext` can _only_ take an existing context, and then we call `setValue` for any of these. Not sure if you have a preference?